ChemistryLib/PhreeqcIO.h

```diff
@@ -41,13 +44,14 @@ class PhreeqcIO
                   process_id_to_component_name_map);
 
     void doWaterChemistryCalculation(
-        std::vector<GlobalVector*>& process_solutions, double const dt);
+        std::vector<GlobalVector*>& process_solutions,
+        double const dt) override;
 
     void setAqueousSolutionsOrUpdateProcessSolutions(
         std::vector<GlobalVector*> const& process_solutions,
         Status const status);
 
-    void setTimeStep(double const dt) { _dt = dt; }
```

Comment:
Not this PR; The `doWaterChemistryCalculation` has `dt` as an argument, and this function also handles `dt`. What's the difference?

Reply:
Dropped `setTimeStep` function.